src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_pix2pix_zero.py

```diff
@@ -542,11 +542,29 @@ def construct_direction(self, embs_source: torch.Tensor, embs_target: torch.Tens
         """Constructs the edit direction to steer the image generation process semantically."""
         return (embs_target.mean(0) - embs_source.mean(0)).unsqueeze(0)
 
+    @torch.no_grad()
+    def get_embeds(self, prompt: List[str]) -> torch.FloatTensor:
+        input_ids = self.tokenizer(
+            prompt,
+            padding="max_length",
+            max_length=self.tokenizer.model_max_length,
+            truncation=True,
+            return_tensors="pt",
+        ).input_ids
+
+        input_ids = input_ids.to(self.text_encoder.device)
+
+        embeds = self.text_encoder(input_ids)[0]
+
+        return embeds.mean(0)[None]
```

Comment:
I like your approach! But do note that the authors note they you need a good number of prompts (the more, the better) to get plausible quality. Cc: @apolinario 

We should note this from the docs, too, so that the users are fully aware.